src/common/secured-property.ts

```diff
@@ -48,7 +48,7 @@ SecuredProperty.descriptionFor = (value: string) => stripIndent`
 @ObjectType({
   description: SecuredProperty.descriptionFor('a string'),
 })
-export abstract class SecuredString extends SecuredProperty<string>(
+export abstract class SecuredString extends SecuredProperty<string | null>(
```

Comment:
Yep you're right, fixing now.

Reply:
@CarsonF so ran into some odd issues here. The `internship-engagement.service` is setting a lot of securedstring values to null intentionally. i.e.:

```
        response.intern = {
          displayFirstName: result.records[0].get('displayFirstName'),
          displayLastName: result.records[0].get('displayLastName'),
          realFirstName: result.records[0].get('realFirstName'),
          realLastName: result.records[0].get('realLastName'),
          email: result.records[0].get('email'),
          id: result.records[0].get('userId'),
          createdAt: DateTime.local(),
          timezone: {
            value: null,
            canRead: true,
            canEdit: true,
          },
          phone: {
            value: null,
            canRead: true,
            canEdit: true,
          },
          bio: {
            value: null,
            canRead: true,
            canEdit: true,
          },
        };
```

easiest fix I can think of is instead of setting value to `null`, set it to `undefined`. At the moment, these are the errors I'm seeing on every `value: null` line:

```
(property) SecuredPropertyClass<string>.value?: string | undefined
Type 'null' is not assignable to type 'string | undefined'.ts(2322)
secured-property.ts(32, 14): The expected type comes from property 'value' which is declared here on type 'SecuredString'
```

Reply:
Those are stubbed responses? I would just change them to `value: undefined` for now

Reply:
Yep, stubbed responses. Just pushed up the update